### DISCUSSION TITLE
cannot reproduce FID of 7.32 using https://huggingface.co/PixArt-alpha/PixArt-alpha/blob/main/PixArt-XL-2-256x256-MSCOCO-FID732.pth

### DISCUSSION THREAD

**Discussion by Gus-Guo** [PixArt-alpha/PixArt-alpha#123] [Q&A]
using cfg_scale=4.5, sampler=dpmsolver, randomly select 30k prompts from captions on coco2014 val split to generate 256x256 images, ckpt from https://huggingface.co/PixArt-alpha/PixArt-alpha/blob/main/PixArt-XL-2-256x256-MSCOCO-FID732.pth, I cannot reproduce FID result of 7.32. reference images are from val split and center crop into 256x256. Is there anything wrong about my config?

**Comment by lawrence-cj** [comment]
So, what's the result you get?

**Reply by Gus-Guo** [reply]
I get a FID score of nearly 14.  I do not know whether is there any thing wrong about my configuration or operation. Do you have any suggestion?

**Reply by lawrence-cj** [reply]
It seems we are using the similar config. I'm not sure what is causing the difference.

**Reply by Gus-Guo** [reply]
could you please share which inception model you was using to compute the FID score?

**Reply by lawrence-cj** [reply]
FID_WEIGHTS_URL = 'https://github.com/mseitzer/pytorch-fid/releases/download/fid_weights/pt_inception-2015-12-05-6726825d.pth'

**Reply by Gus-Guo** [reply]
we use the same model...so confused why I can't get FID of 7.32...

**Reply by lawrence-cj** [reply]
I'll check if the ckpt is the correct one when I'm free. Busy with other issues

**Reply by Gus-Guo** [reply]
got it. Thank you very much.

**Reply by Gus-Guo** [reply]
@lawrence-cj Hi lawrence, do you have plan to disclose your FID measurement code?

**Reply by lawrence-cj** [reply]
Will update here. But no exact date.
https://github.com/PixArt-alpha/PixArt-Eval
